vumi/transports/dmark/dmark_ussd.py

```diff
@@ -113,8 +113,20 @@ def handle_raw_inbound_message(self, request_id, request):
             log.msg('Unhappy incoming message: %r' % (errors,))
             self.finish_request(
                 request_id, json.dumps(errors), code=http.BAD_REQUEST)
+            yield self.add_status(
+                component='request',
```

Comment:
Can't remember if this is a thing that we discussed. Did we decide on separate request and response components? Wondering if it would make more sense to have an `http` component. Thoughts?